tsparser/src/builder/transpiler.rs

```diff
@@ -116,8 +115,8 @@ impl OutputTranspiler for EsbuildCompiler<'_> {
                 ];
 
                 match p.debug {
-                    1 => command.push("--inspect".to_string()),
-                    2 => command.push("--inspect-brk".to_string()),
+                    DebugMode::Enabled => command.push("--inspect".to_string()),
+                    DebugMode::Break => command.push("--inspect-brk".to_string()),
                     _ => {}
```

Comment:
Remove the catch-all in favor of only handling the Disabled case explicitly. That way we’ll get a compiler error here if we add more enum cases